README.md

```diff
@@ -1 +1,42 @@
-# Awesome-book-ES6
\ No newline at end of file
+# Awsome-book-using-ES6
+# capstone project
+## Built With
+
+- HTML
+- CSS
+- javadcript
+
+## contributor
+
+## live Demo
+Feel free to check the: [link](https://richez003.github.io/Awsome-book-using-ES6/).
+
+## Author
+👤 **Richard Friday**
+
+- GitHub: [@Richez003](https://github.com/Richez003)
+- Twitter: [@richardfriday14](https://twitter.com/richardfriday14)
+- LinkedIn: [@LinkedIn](https://www.linkedin.com/in/richard-friday-54980718a)
+
+
+## 🤝 Contributing
+👤 **Kayonga Chiteta**
+
+- Twitter: [@KayongaChiteta3](https://twitter.com/KayongaChiteta3
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] This link is broken. Please make sure to fix it.

Reply:
removed